trade-finance-manager-ui/templates/case/tasks/task.njk

```diff
@@ -4,11 +4,15 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle -%}Deal {{ deal.details.ukefDealId }} - Task {{ task.title }}{%- endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.